src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Dec2Hex.cs

```diff
@@ -0,0 +1,147 @@
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.Dec2HexArg1, TexlStrings.Dec2HexArg2 };
+        }
+    }
+
+    // Dec2HexT(number:[n], [places:n])
+    internal sealed class Dec2HexTFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool IsStateless => true;
+
+        public override bool SupportsParamCoercion => true;
+
+        public override bool HasPreciseErrors => true;
+
+        public override bool CheckTypesAndSemanticsOnly => true;
+
+        private static readonly DType TabularReturnType = DType.CreateTable(new TypedName(DType.String, ColumnName_Value));
+
+        public Dec2HexTFunction()
+            : base("Dec2Hex", TexlStrings.AboutDec2HexT, FunctionCategories.Table, TabularReturnType, 0, 1, 2)
+        {
+        }
+
+        public override IEnumerable<TexlStrings.StringGetter[]> GetSignatures()
+        {
+            yield return new[] { TexlStrings.Dec2HexTArg1, TexlStrings.Dec2HexTArg2 };
+        }
+
+        protected override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        {
+            Contracts.AssertValue(args);
+            Contracts.AssertAllValues(args);
+            Contracts.AssertValue(argTypes);
+            Contracts.Assert(args.Length == argTypes.Length);
+            Contracts.AssertValue(errors);
+            Contracts.Assert(MinArity <= args.Length && args.Length <= MaxArity);
+
+            var fValid = base.CheckTypes(context, args, argTypes, errors, out returnType, out nodeToCoercedTypeMap);
+            if (argTypes.Length == 1)
+            {
+                var type = argTypes[0];
+                var arg = args[0];
+                if (!type.IsTable)
+                {
+                    errors.EnsureError(DocumentErrorSeverity.Severe, arg, TexlStrings.ErrTypeError);
+                    return false;
+                }
+            }
+            else if (argTypes.Length == 2)
+            {
+                var type0 = argTypes[0];
+                var type1 = argTypes[1];
+                var arg1 = args[0];
+                var arg2 = args[0];
+
+                if (!type0.IsTable && !type1.IsTable)
+                {
+                    errors.EnsureError(DocumentErrorSeverity.Severe, arg1, TexlStrings.ErrTypeError);
+                    errors.EnsureError(DocumentErrorSeverity.Severe, arg2, TexlStrings.ErrTypeError);
+                    return false;
+                }
+
+                var otherType = DType.Invalid;
+                TexlNode otherArg = null;
+
+                returnType = DType.CreateTable(new TypedName(DType.String, new DName(ColumnName_ValueStr)));
```

Comment:
>returnType = DType.CreateTable(new TypedName(DType.String, new DName(ColumnName_ValueStr))); [](http://example.com/codeflow?start=16&length=92)

You can probably remove this, since this is set at line 72 at the base CheckTypes implementation